docker-compose.fvm.yml

```diff
@@ -15,6 +15,7 @@ services:
     platform: linux/amd64
     command: npm run node
     tty: true
+    restart: always
```

Comment:
Let's do it just on productions for now we have to make sure this is really crash

Reply:
ok kub